README.md

```diff
@@ -461,8 +461,10 @@ zoomer: {
     // canvas
     residueFont: "13px Helvetica Neue",
     residueFontOffset: [0, 0], // horizontal and vertical offset of residue font in px
+    maxResidueCodeLength: 6, // maximum length of residue code to be displayed in the viewer, codes with greater length are truncated with a fading effect 
     canvasEventScale: 1,
-    residueFontGetter: undefined // this can be overriden with a custom function of the following signature: (char: string, pos: {x: number, y: number}) => { font: string, color: string}
+    residueFontPropsGetter: undefined // this can be overriden with a custom function of the following signature: (residueCode: string, pos: {x: number, y: number}) => { font: string, color: string}
+    residueTilePropsGetter: undefined // function to get the properties for a specific residue tile, which can be overriden with a custom function of the following signature: (pos: {x: number, y: number}) => { hasBottomBorder: boolean }
```

Comment:
Should I remove this option, which I exposed for controlling tile characteristics, as we no longer require to modify them as per the new Product/UX requirements (no need of a bottom bar for non-natural monomers)?

Reply:
yeah we should remove it if the requirement is final